O/OpenModelica/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+using BinaryBuilder, Pkg
+
+name = "OpenModelica"
+version = v"1.23.0"
+
+sources = [
+   GitSource("https://github.com/OpenModelica/OpenModelica",
```

Comment:
Not sure this is related to your issue, but I seem to remember there are some assumptions somewhere that the repo URL should end with `.git`:
```suggestion
   GitSource("https://github.com/OpenModelica/OpenModelica.git",
```

Reply:
Still getting the same error with git submodules. I see other Yggdrasil repos successfully doing git submodules - so puzzling why this one is having a challenge.